sql/dql/aggregate-queries/distinct-clause.md

```diff
@@ -92,21 +92,21 @@ FROM pokedex_name;
 
 ## Revision
 
-Consider the following table and the following query. Fill the gaps such that we will have no duplicates in the resulting table:
+Consider the following table named GRADES with the following query. Fill the gaps such that we will have no duplicate NAME's in the resulting table:
```

Comment:
Hey @rettal,

Thanks for taking the time to improve this question!

I have a couple of minor suggestions:
- we should enclose the table & column names in backticks
- we should remove the possessive (NAME's) & change it to plural (NAMEs)

```suggestion
Consider the following table named `GRADES` with the following query. Fill the gaps such that we will have no duplicate `NAME`s in the resulting table:
```